pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
```

Comment:
For this classLoader: https://github.com/arquillian/arquillian-container-chameleon/pull/68/files#diff-ef6955c1e4ea3048fd0e4eaf122fa43bR20. in configuration module.